aten/src/ATen/mps/MPSAllocator.h

```diff
@@ -9,27 +10,10 @@
 
 // this implementation is based on CUDACachingAllocator.
 // It utilizes Metal Heaps to improve the performance with buffer allocation.
+// Do not include this header. Use MPSAllocatorInterface.h instead.
```

Comment:
Unclear the need for this comment here? Was it being used before ?

Reply:
Yes, this is a private header, it was being included in View.mm, OperationUtils.mm, Unique.mm, and etc. The public interface MPSAllocatorInterface.h should be used instead.